### ISSUE TITLE
Create a new, empty Node.js project

### ISSUE THREAD

**Issue by mgspross** [jalawelo/math-problem-generator#9]
This wasn't fully fleshed out earlier, but before we continue with work on issue #6, we need to first create a new, empty Node.js project (without any code) so that @newoga and I (@mgspross) can start working on unit tests for the program while work continues in parallel on the JavaScript conversion (@franchet and @Icarus-Dragon).

Specifically, this is what we need done for this issue (no more, no less):

- Create a new Node.js project with the Node Package Manager (`npm`), which is a commonly-used tool in the Node.js toolchain (Google is your friend here, but feel free ask questions if needed)
- Commit and push the files that `npm` creates in a new pull request to the `master` branch in this repository. **To be clear, there will no additional code included in this PR**: we only need/want the empty project at this point so we can all be on the same page

After the Node.js project is up, we'll want to do the JavaScript conversion (#6) in that project, since our goal is to create a Node.js command-line app.

@franchet: we're thinking you can take this task on, but have @Icarus-Dragon with you when you work on this so you can both get exposure to how to create a new Node project from scratch. Leave a comment to let us know and we'll be able to assign this issue to you at that point.

**Comment by Icarus-Dragon** [comment]
you want me to create the server in node.js file










Delanoxbannister11@gmail.com
<Delanoxbannister11@gmail.com>

**Comment by newoga** [comment]
@Icarus-Dragon Yes, feel free to give this a try. Look into `npm init` documentation.

**Comment by Icarus-Dragon** [comment]
thank you. i will give it a go










Delanoxbannister11@gmail.com
<Delanoxbannister11@gmail.com>

**Comment by Icarus-Dragon** [comment]
I upload the empty node.js project. Just wanted to know if you saw it

**Comment by mgspross** [comment]
@Icarus-Dragon: I took a look at what you put up. It's not quite what we're looking for. Please review the following notes for how to proceed:

- As @newoga suggested, look into `npm init`, which is the tool used to start a new Node project
- There shouldn't be any `.js` files yet. When you run `npm init`, it should generate a single `.json` file, which is the file we need
- Remember to create a pull request when submitting code changes meant for this repository, rather than placing code in your own personal GitHub repository, otherwise we won't be able to incorporate your contributions into the project here. Please refer to [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) for more details.

**Comment by Icarus-Dragon** [comment]
I created it.
gonna put it on github

**Comment by Icarus-Dragon** [comment]
Can you see my push request

**Comment by mgspross** [comment]
This was fixed and merged into `master`.
